setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+setup(
+    name='mimic',
+    version="0.0.1",
+    packages=find_packages(),
+    install_requires=['docutils'],
```

Comment:
Yes and no. ``install_requires`` lists which packages ("egg"s or "wheel"s) should be installed and requirements.txt lists specific versions. So if, for example, one required at least version 1.3 of docutils, one has
```
install_requires=['docutils']
```
in setup.py and ``docutils>=1.3`` in ``requirements.txt``.

In this specific case I don't think we need anything other than a fairly non-ancient docutils so it'd be safe to leave ``requirements.txt`` empty or delete it. It's mostly a question of personal taste.